src/EditorFeatures/TestUtilities2/Intellisense/TestStateBase.vb

```diff
@@ -213,17 +213,9 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
 
         Public MustOverride Overloads Function AssertCompletionSession(Optional projectionsView As ITextView = Nothing) As Task
 
-        Public Sub AssertCompletionItemsContainAll(displayText As String())
```

Comment:
💭 I'm not a big fan of this piece of the change.

Reply:
What specifically do not you like here?
Moving from Sub to Async? Removing Assert from Modern implementation? 

---
In reply to: [314786506](https://github.com/dotnet/roslyn/pull/38021#discussion_r314786506) [](ancestors = 314786506)